eslint/babel-eslint-parser/src/worker/maybeParse.cjs

```diff
@@ -10,7 +10,7 @@ const MULTIPLE_OVERRIDES = /More than one plugin attempted to override parsing/;
 
 module.exports = function maybeParse(code, options) {
   if (!extractParserOptionsConfigItem) {
-    extractParserOptionsConfigItem = babel.createConfigItem(
+    extractParserOptionsConfigItem = babel.createConfigItemSync(
```

Comment:
Doesn't this break compatibility with older `@babel/core`?

Reply:
That Babel core is imported from babel-core.cjs, where compatibility is provided for 7.11.